### DISCUSSION TITLE
Commands (in project dependencies) not found after `npm install`

### DISCUSSION THREAD

**Discussion by chris-dura** [ohmyzsh/ohmyzsh#10037] [Help]
I set up a new Macbook (macOS Catalina), awhile ago with a "standard" setup that I've documented step-by-step.
However, unlike previous times that I've set up OMZ, afaik, I'm _not_ "coming from bash", as `zsh` is the default shell on macOS, now?

Anyway, I'm usually running commands via `npm scripts`, so I didn't notice the issue until now, but after an `npm install`, I cannot run certain commands _directly_, but they work fine if I use an `npm run <script>`, or if I use `npx`...

```sh
npm install cpy-cli

# This works
npx cpy --version

# Or, this works if there is an npm script named `copy-version` that runs: `cpy --version`)
npm run copy-version

# This does _not_ work... but it did in my previous OMZ environments
cpy --version
```

I'm also using NVM, and the `npm` and `nvm` commands work fine...

```sh
# Both these work
npm --version
nvm list
```

I see no obvious differences in my old `~/.zshrc` and this new `~/.zshrc`.

----

TL;DR for set up steps...

1. Install NVM
```sh
cd ~
touch .zshrc
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
```
2. Install Node
```sh
nvm install --lts
```
3. Update NPM
```sh
npm install npm@latest -g
```
4. Install Oh My Zsh
```sh
sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)" --keep-zshrc
```
